libs/ui-components/src/components/form/TextListField.tsx

```
@@ -0,0 +1,88 @@
+};
+
+const maxWidth = '16ch';
+
+const TextListField = ({ name, onChangeCallback, addButtonText, focusAddOnStart, helperText }: TextListFieldProps) => {
+  const [{ value: valueList }, meta, { setValue: setValueList, setTouched }] = useField<string[]>(name);
+  const updateValueList = async (newValueList: string[]) => {
+    const errors = await setValueList(newValueList, true);
+    const hasErrors = Object.keys(errors || {}).length > 0;
+
+    setTouched(true);
+    onChangeCallback && onChangeCallback(newValueList, hasErrors);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)